server/controllers/routes/user/student/getAlumniCohort.js

```diff
@@ -0,0 +1,27 @@
+const { getSpecificAlumnit } = require('../../../../database/queries');
+
+const getAlumniCohort = async (req, res, next) => {
+  try {
+    const { cohortId } = req.params;
+    if (cohortId > 0) {
+      const { rows } = await getSpecificAlumnit(cohortId);
+      const data = { ...rows[0] };
```

Comment:
The expected response is an array of students objects not only the first object so you need to clone all rows
```suggestion
      const data = [...rows];
```